android/src/main/kotlin/de/mintware/barcode_scan/BarcodeScannerActivity.kt

```diff
@@ -78,6 +79,10 @@ class BarcodeScannerActivity : Activity(), ZXingScannerView.ResultHandler {
         }
         val item = menu.add(0, TOGGLE_FLASH, 0, buttonText)
         item.setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS)
+
+        val item = menu.add(0, CANCEL, 0, config.stringsMap["cancel"])
```

Comment:
Duplicate declaration of the item variable. Rename this to `cancelButton` and the one above `flashButton`